test/test_torch.py

```diff
@@ -15440,7 +15440,8 @@ def test_orgqr_errors(self, device):
 
     @skipCUDAIfNoMagma
     @skipCPUIfNoLapack
-    def test_lu(self, device):
+    @dtypes(torch.double, torch.complex64, torch.complex128)
```

Comment:
nit - let's use the alias `torch.cfloat`, `torch.cdouble` since that's what we have been using for most other tests